ENH: permit latex inputs w/ docutils builds (#787)
scipy-conference/scipy_proceedings: README.md

Reviewer: Do we want to merge 2023 branch changes of README.md to dev? There may be some language changes (besides just dates) there that aren't in dev branch.

Please submit your papers by 23:59 PST of the *1st Draft for Submission*
Deadline.

> Submit your papers as an ReStructured Text (rst) or LaTeX file via PR against this repository. Supporting LaTeX submissions is very new this year, so please consider it to be in beta, and please only use this option if you are already familiar with writing papers in LaTeX.

Reviewer: ```suggestion
Submit your papers as a reStructuredText (rst) or LaTeX file via PR against this repository. Supporting LaTeX submissions is very new this year, so please consider it to be in beta, and please only use this option if you are already familiar with writing papers in LaTeX.
```
